tests/TestUtil.elm

```diff
@@ -25,6 +25,25 @@ stringToIntermediate str =
             Nothing
 
 
+{-| Removes indentation from a multi-line string.
+-}
+dedent : Int -> String -> String
```

Comment:
Specifying the prefix manually is error prone in practice (_e.g._ imagine if you shuffled, restructured and re-indented the tests -- you would need to change the prefix manually). 

It's more robust to figure out the common prefix from the lines, see for example: https://github.com/python/cpython/blob/17473347942353946fe455f797a2197cb89c1090/Lib/textwrap.py#L414

When implementing dedent, make sure you ignore lines which are completely empty:

```
    some text
(empty line here)
    some other text
```